app_feup/lib/view/Pages/login_page_view.dart

```diff
@@ -153,6 +155,33 @@ class _LoginPageViewState extends State<LoginPageView> {
     );
   }
 
+  /// Creates the widget for the user to choose their faculty
+  Widget createFacultyInput() {
+    final faculties = ['faup', 'fbaup', 'fcup', 'fcnaup', 'fadeup', 'fdup',
```

Comment:
The name of the faculties will be important for several parts. Can you add this to the `constants.dart` file?